### PR TITLE
Shift from PhantomJS to Chromium to get accessibility tests working.

### PR DESCRIPTION
**Pull request by ndouglas**
## Description

I've just been messing around in a live Tugboat environment, so this might not work for a while.

## QA steps

As user _uid_ with _user_role_
1. Do this
1. Then that
1. Then validate Acceptance Criteria from issue
- [ ] This
- [ ] That
- [ ] The other thing

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [x] `Core Application Team`
- [ ] `Product Support Team`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change? 
- [ ] Yes, and it's written in issue ____ and queued for publication. 
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written 
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [ ] No announcement is needed for this code change. 
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -408,6 +408,7 @@
             "npm run stylelint-themes"
         ],
         "va:test:accessibility": [
+            "apt-get install chromium",
```
Comment (ElijahLynn) [inline composer.json:411]: We will need to do this at the OS level stages for Lando, Tugboat and BRD (yum).

Comment (indytechcook) [inline composer.json:411]: Move this to the tugboat `config.yml` init hook.

Comment (ndouglas) [reply composer.json:411]: yeah, I [did that initially](https://github.com/department-of-veterans-affairs/va.gov-cms-test/pull/183/commits/f1184eba4aea78cc34bb1b59f8d2ce911f274e0a), just moved it here for faster turnaround while trying to figure out what worked and what didn't

**Comment by ElijahLynn** [comment]
Closing as we did this with cypress-axe now.

**Pull request closed without merging**